### ISSUE TITLE
[py2] Importing from wx backend after selecting qt4 backend!

### ISSUE THREAD

**Issue by ignatenkobrain** [enthought/pyface#180]
``` python
======================================================================
ERROR: Failure: RuntimeError (Importing from wx backend after selecting qt4 backend!)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/builddir/build/BUILD/pyface-5.0.0/build/lib/pyface/ui/wx/grid/tests/composite_grid_model_test_case.py", line 4, in <module>
    from pyface.ui.wx.grid.api \
  File "/builddir/build/BUILD/pyface-5.0.0/build/lib/pyface/ui/wx/grid/api.py", line 16, in <module>
    from .grid import Grid
  File "/builddir/build/BUILD/pyface-5.0.0/build/lib/pyface/ui/wx/grid/grid.py", line 38, in <module>
    from traitsui.wx.dnd_editor import FileDropSource
  File "/usr/lib/python2.7/site-packages/traitsui/wx/__init__.py", line 26, in <module>
    import toolkit
  File "/usr/lib/python2.7/site-packages/traitsui/wx/toolkit.py", line 28, in <module>
    assert_toolkit_import('wx')
  File "/usr/lib/python2.7/site-packages/traitsui/toolkit.py", line 60, in assert_toolkit_import
    "backend!" % (name, ETSConfig.toolkit))
RuntimeError: Importing from wx backend after selecting qt4 backend!

======================================================================
ERROR: Failure: RuntimeError (Importing from wx backend after selecting qt4 backend!)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/nose/loader.py", line 418, in loadTestsFromName
    addr.filename, addr.module)
  File "/usr/lib/python2.7/site-packages/nose/importer.py", line 47, in importFromPath
    return self.importFromDir(dir_path, fqname)
  File "/usr/lib/python2.7/site-packages/nose/importer.py", line 94, in importFromDir
    mod = load_module(part_fqname, fh, filename, desc)
  File "/builddir/build/BUILD/pyface-5.0.0/build/lib/pyface/ui/wx/grid/tests/simple_grid_model_test_case.py", line 4, in <module>
    from pyface.ui.wx.grid.api \
  File "/builddir/build/BUILD/pyface-5.0.0/build/lib/pyface/ui/wx/grid/api.py", line 16, in <module>
    from .grid import Grid
  File "/builddir/build/BUILD/pyface-5.0.0/build/lib/pyface/ui/wx/grid/grid.py", line 38, in <module>
    from traitsui.wx.dnd_editor import FileDropSource
  File "/usr/lib/python2.7/site-packages/traitsui/wx/__init__.py", line 26, in <module>
    import toolkit
  File "/usr/lib/python2.7/site-packages/traitsui/wx/toolkit.py", line 28, in <module>
    assert_toolkit_import('wx')
  File "/usr/lib/python2.7/site-packages/traitsui/toolkit.py", line 60, in assert_toolkit_import
    "backend!" % (name, ETSConfig.toolkit))
RuntimeError: Importing from wx backend after selecting qt4 backend!

----------------------------------------------------------------------
```

**Comment by ignatenkobrain** [comment]
Happens only with python 2

**Comment by ignatenkobrain** [comment]
I just ran tests. Nothing serious

**Comment by fbender** [comment]
Would you please provide a test case then? It'll be way easier to sort this out when it's reproducible.

**Comment by corranwebster** [comment]
This should be resolved by #105.
